src/unitxt/operator.py

```diff
@@ -508,3 +508,40 @@ def process(self, *args, **kwargs) -> MultiStream:
         for operator in self.steps[1 : self._get_max_steps()]:
             multi_stream = operator(multi_stream)
         return multi_stream
+
+
+class PlugInOperator(StreamInstanceOperator):
+    """Applies an operator to each instance in a stream based on specific field.
+
+    Args:
+        field (str): name of the field that contains a single name of the operator to be applied
+        for the processing of the instance. Each operator is equipped with 'process_instance()'
+        method.
+
+        default_operator (List[str]): A list of default operators to be used if no operators are found in the instance.
+
+    Example:
```

Comment:
This is example is really not clear to me. Why is "processors.to_string" added to field "c"?